packages/react/index.ts

```diff
@@ -355,7 +355,7 @@ function reactAxe(
   timeout = _timeout;
   context = _context;
 
-  const runOnly = conf['runOnly'];
+  const runOnly = (conf || {})['runOnly'];
```

Comment:
We could alternately define a default argument of `{}` for `conf` on line 350, but I wasn't sure if the config call below would change with an empty object vs. skipping the call to `configure` completely.